connector_search_engine/models/se_index.py

```diff
@@ -11,6 +11,20 @@ class SeIndex(models.Model):
     _name = 'se.index'
     _description = 'Se Index'
 
+    @api.model
+    def _get_model_domain(self):
+        models = self.env['ir.model'].search([('transient', '=', False)])
+        se_model_ids = []
+        for model in models:
+            if model.model == 'se.binding':
+                continue
+            try:
+                if self.env[model.model]._se_model:
```

Comment:
You can use hasattr(self.env[model.model], '_se_model') and self.env[model.model]._se_model instead of the try / except

Reply:
Strange, I was sure I had fix that some days ago...
Maybe I forgot to push

Reply:
I had forgotten to push indeed... Fixed now